src/utils/routes/index.ts

```diff
@@ -0,0 +1,4 @@
+// Remove : from slug
+const slugToVariable = (slug: string) => slug.replace(":", "");
+
+export { slugToVariable };
```

Comment:
This could potentially be deleted, as I don't see it being used elsewhere in the code